### PR TITLE
16 narrative

### LINKED ISSUE #41: Add Narrative for Landing Page
Based on #16 - Sub Issue

See: https://fhir.hl7.at/r4-core-main/index.html 
Add Information:
- What is the hl7-at-core IG, 
- Why do you need it, 
- what resources are generally (not) in there
- Usage -> mostly to use in your own austrian IGs
- Aligment with ELGA

### LINKED ISSUE #42: Add Patient Narrative
Based on #16 - Sub Issue

See https://fhir.hl7.at/r4-core-main/StructureDefinition-at-core-patient.html 

Examples how to do better: 
http://build.fhir.org/ig/hl7au/au-fhir-childhealth/StructureDefinition-ncdhc-patient-baby.html

http://build.fhir.org/ig/hl7ch/ch-core/StructureDefinition-ch-core-patient.html

### PR DESCRIPTION
**Pull request by gabriel0316**
Closes #41 
Closes #42

### CODE REVIEW

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+- This guide provides a set of sample Code Systems and ValueSets based on FSH files. Note, that Code Systems and ValueSets will generally be made available in a separate IG based on [TerminoloGit](https://gitlab.com/elga-gmbh/termgit).
 
-For the creation of this implementation guide FHIR® Shorthand was used.
+This implementation guide contains only those profiles and extension where adaptions for Austria were necessary.
 
-[HL7® Confluence site](https://confluence.hl7.org/display/FHIRI/FHIR+Shorthand)
+### Relation to Austrian EHR (ELGA)
 
-[FHIR® Shorthand Documentation](https://build.fhir.org/ig/HL7/fhir-shorthand) 
+Currently, the Austrian EHR (ELGA) is mainly based on HL7® CDA®. In order to enable the creation of FHIR® resources that have a similar data structure this Implementation Guide provides mappings to the CDA® based implementation guides (e.g. [Mappings for Allgemeiner Implementierungsleitfaden v3](StructureDefinition-at-core-patient-mappings.html#mappings-for-allgemeiner-implementierungsleitfaden-v3-https-wiki)).
```
Comment (oliver-krauss) [inline input/pagecontent/index.md:16]: TODO: Discuss in FHIR Call if we want to provide this. According to @Nik-xT  ~3h work.

File: input/fsh/at-core-patient.fsh [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+# Created by: HL7® Austria, TC FHIR®
 ##############################################################################*/
 
 Profile:        HL7ATCorePatient
 Parent:         Patient
 Id:             at-core-patient
 Title:          "HL7® AT Core Patient Profile"
-Description:    "HL7® Austria FHIR® Core Profile for patient pata in Austria."
+Description:    "HL7® Austria FHIR® Core Profile for patient data in Austria.
+The HL7 AT Core Patient is based upon the core FHIR Patient Resource and designed to meet the applicable patient demographic data elements in Austria. It identifies which core elements, extensions, vocabularies and value sets SHALL be present in the resource when using this profile."
```
Comment (oliver-krauss) [inline input/fsh/at-core-patient.fsh:12]: Maybe add this sentence: "Note, this extension represents the common structure of address information within Austrian information systems."

Also -> FHIR and HL7 are both missing ®

**Review by oliver-krauss — CHANGES_REQUESTED** [review]
see comments

File: input/pagecontent/index.md (hunk 1 above)
Comment (aschuler84) [reply input/pagecontent/index.md:16]: Will be covered via #6 and #7

File: input/fsh/at-core-patient.fsh (hunk 2 above)
Comment (gabriel0316) [reply input/fsh/at-core-patient.fsh]: Sentence and ® have been added.

**Review by oliver-krauss — APPROVED** [review]

**Review by aschuler84 — APPROVED** [review]

**Pull request merged**